.github/workflows/stale.yml

```diff
@@ -25,3 +25,7 @@ jobs:
         stale-pr-message: 'This PR is stale because it has been open for 60 days with no activity. Please update the PR or it will be closed in 7 days.'
```

Comment:
This should be updated to say "180" for PRs based on the config below

Reply:
Good call, fixed